src/statusUpdate.js

```diff
@@ -0,0 +1,11 @@
+// function to mark as completed
+
+const markComplete = (tasks, index) => {
+  if (tasks[index].solved === false) {
+    tasks[index].solved = true;
+  } else {
+    tasks[index].solved = false;
+  }
+  return tasks[index].solved;
+};
```

Comment:
- [ ] When I reload the page the `checked` boxes become `unchecked`. Please make sure that the updated `array` of items is stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored.